text/0004-channel-participation-api-without-system-channel.md

```diff
@@ -50,14 +50,21 @@ starts by scanning the system channel in order to discover existing channels and
 channels increase, this scan prolongs the process of adding a new orderer.  
 
 ## Operational problems
-There are several operational problems that affect large scale deployment and hinder efficient management of cloud
-environments using automatic tooling.
+The dependence on a system channel causes channel management to become unnecessarily fragile and complicated, impacting
+large scale deployment and hindering efficient management of cloud environments using automatic tooling.
+
+* It is not possible to spin up ordering service nodes without first coordinating on a system channel genesis block.
 
 * Channel creation (using the system channel) utilizes information contained in the system channel - e.g. MSP information
-of the organization - in order to apply it to the newly created channel. This creates the convoluted situation in
+of the organization - in order to apply it to the newly created channel. This creates an unnecessary and convoluted situation in
 which an organization that wants to update its details in the consortium must coordinate that action with third parties
 that have write privileges to the system channel.
 
+* Updating the system channel has been a challenge for users because it works differently than application channels
+(MSP definitions are in different paths, the config structure is slightly different, etc).
+
+* The dependence on the system channel makes creating a channel unnecessarily complex and fragile, for example the need to pick subsets of consenters.
```

Comment:
for example, the channel creation transaction generation must assume the current state of system channel and the template channel configuration it will create.  This leads to relatively simple use cases like specifying a subset of consenters at channel creation being difficult and error prone.